src/crypto/__tests__/crypto-alpha-38-test.ts

```diff
@@ -35,9 +35,9 @@ beforeAll(() => {
 // Test for:
 c.typedMessageParse && c.typedMessageStringify
 test('Crypto alpha v38 Typed Message', () => {
-    const textWith0Meta = makeTypedMessage('text message1')
-    const textWith1Meta = makeTypedMessage('text message', new Map([['MetadataKey', 'MetadataValue']]))
-    const textWith2Meta = makeTypedMessage(
+    const textWith0Meta = makeTypedMessageText('text message1')
```

Comment:
why not definition an array

```ts
const examples = [
  [
    [stringify, parsed],
    [stringify, parsed],
    [stringify, parsed],
  ]
]
```

Reply:
This is not the primary goal of this PR. >_<